webdriver/tests/find_element_from_shadow_root/find.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from tests.support.asserts import assert_success
+
+
+def get_shadow_root(session, element):
+    return session.transport.send(
+        "GET", "session/{session_id}/element/{element_id}/shadow".format(
+            session_id=session.session_id,
+            element_id=element.id))
+
+
+def find_element(session, shadow_root, using, value):
+    return session.transport.send(
+        "POST", "session/{session_id}/shadow/{shadow_id}/element".format(
+            session_id=session.session_id,
+            shadow_id=shadow_root.id),
+        {"using": using, "value": value})
+
+
+@pytest.fixture
+def get_checkbox_dom(inline):
```

Comment:
Please move this fixture into a `conftest.py` file in that directory. For `performActions` we named the fixture for the test page `test_actions_page`. Maybe we can do it similar here.

We could also add this HTML file under `support/html/shadow_dom.html`, so that we can share it between all the different shadow DOM related APIs.

Reply:
Done in 94b8624